tt_metal/include/compute_kernel_api/mask.h

```diff
@@ -16,12 +15,14 @@
 #define MATH(x)
 #endif
 
-
-
 namespace ckernel {
 
 ALWI void mask_tile_init() {
-    MATH(( llk_math_eltwise_unary_sfpu_mask_init<true>() )); // TODO(AP): move out init
+    MATH((llk_math_eltwise_unary_sfpu_mask_init<true>()));  // TODO(AP): move out init
+}
+
+ALWI void mask_posinf_tile(uint32_t idst_data, uint32_t idst2_mask) {
```

Comment:
Documentation for this function?

Reply:
It only replaces the fill value from `0` in the original `mask_tile` to `inf`. I will add the doc asap.